app/modules/welcome/components/landing_screen.svelte

```diff
@@ -0,0 +1,188 @@
+<script>
+  import { i18n, I18n } from '#user/lib/i18n'
+  import { icon } from '#lib/utils'
+
+  const needNameExplanation = app.user.lang !== 'fr'
+  const { loggedIn } = app.user
+</script>
+
+<section id="landingScreen" class="text-center">
+  <div class="name">
+    <h2 class="respect-case">
+      inventaire
+      {#if needNameExplanation}
+        <span class="name-explaination">*{i18n('means "inventory" in French')}</span>
+      {/if}
+    </h2>
+  </div>
+  <!-- wrap the rest in the bottom div to push it down -->
+  <div class="bottom">
+    <ul class="pitch">
+      <li>
+        {@html icon('list')}
+        <h3>{I18n('keep an inventory of your books')}</h3>
+        <p>{i18n('and of what can be done with them:')} "<em>{i18n('this book I can lend, this book I can give, this one is for sale...')}</em>"</p>
+      </li>
+      <li>
+        {@html icon('users')}
+        <h3>{I18n('share it with your friends and communities')}</h3>
+        <p>{i18n('Always dreamed of a collaborative library for your coworking / association / book club? Here it is!')}</p>
+      </li>
+      <li>
+        {@html icon('exchange')}
+        <h3>{I18n('Remember the books you lend or borrow')}</h3>
+        <p>{i18n('The books you lend might actually come back now!')}</p>
```

Comment:
I invoke a change i18n key token (!) to modify the third box `exchange`. This change could be justified by the fact that those texts might be the first and most read words when arriving for the first time on inventaire, the importance of coining some precise words here is quite high. Also, there are more features now than when this `exchange` part was written in 2015, which makes it quite secondary compared to other features from old ones (groups, data editor) to new ones (listings). 

So, I would propose something about the bibliographical data. 

It could be something like `Contribute to an open library` with a Wikdata mention maybe, or more on the book information one can get though inventaire: 
  `Find ideas for the next book you will read`, 
  `Browse users inventories and book lists`. 

If we are open to i18n key changes, then the second box `users` could mention the term `group` like 
  `Make friends, create groups and exchange books together` (which would then wrap up the exchange part).

If this is blocking then i can bury this suggestion in an issue that will be forgotten later.

Reply:
I would indeed prefer to have the rewording in a separate PR, as it is a subject on which we might be nitpicking for a while, blocking the rest of this rewrite, and further developments on this layout. But I do think it's time to start to consider begining a process of getting into rethinking that first screen ([and the time to do it is very soon](https://www.youtube.com/watch?v=l23_-1v__Fg))